docs/command-line-interface/topaz-cli/configuration.mdx

```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
 ```
 Flags:
   -h, --help                         Show context-sensitive help.
-      --no-check                     disable local container running check ($TOPAZ_NO_CHECK)
+  -N, --no-check                     disable local container status check ($TOPAZ_NO_CHECK)
 
   -n, --policy-name=STRING           policy name
   -l, --local-policy-image=STRING    local policy image name
   -r, --resource=STRING              resource url
   -p, --stdout                       generated configuration is printed to stdout but not saved
   -d, --edge-directory               enable edge directory
+  -f, --force                        forcefully create configuration
+  -c, --config="config.yaml"         use topaz configuration file
```

Comment:
I'd just say "topaz configuration file" (without "use"). Similar to the other options like `-n`, `-r`, etc.